Essentials/src/com/earth2me/essentials/craftbukkit/InventoryWorkaround.java

```diff
@@ -267,4 +267,17 @@ public static void setItemInOffHand(final Player p, final ItemStack item) {
             }
         }
     }
+
+    public static int clearItemInOffHand(final Player p, final ItemStack item) {
+        if (hasMainHandSupport) {
+            int removedAmount = 0;
+            if (p.getInventory().getItemInOffHand().getType().equals(item.getType())) {
+                removedAmount = p.getInventory().getItemInOffHand().getAmount();
+                p.getInventory().setItemInOffHand(null);
+            }
+            hasMainHandSupport = true;
```

Comment:
This is redundant?

Reply:
Forgot to remove that when I removed `hasMainHandSupport == null`. Will fix.